src/test/java/pl/touk/sputnik/processor/sonar/SonarProcessorTest.java

```diff
@@ -16,10 +16,11 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class SonarProcessorTest extends TestEnvironment {
+class SonarProcessorTest extends TestEnvironment {
 
     @Test
-    public void shouldFilterResultFiles() {
+    void shouldFilterResultFiles() {
+        //given
```

Comment:
why do you add this comments? You didn't do this to many tests above. IMO It's better to not have those comments at all

Reply:
Historically there were added by developers. With release 2.0 I want to remove them too. I will do it in next commits.